website/docs/migrate-v4-to-v5.mdx

```diff
@@ -0,0 +1,25 @@
+---
+id: migrate-v4-to-v5
+title: Migrating to v5
+---
+
+## iOS
+
+No breaking changes :relieved:
+
+## Android
+
+[v5 introduces a change](https://github.com/react-native-community/react-native-share/issues/936) to remove the `WRITE_EXTERNAL_STORAGE` permission from being automatically added to implementing apps.
+
```

Comment:
```suggestion

This is motivated by the Android 11 system enforcing new restrictions on storage, and mandating the use of scoped storage. If this impacts you, you should read more here to make sure your app will still work on Android 11: [Scoped Storage restrictions](https://developer.android.com/about/versions/11/privacy/storage)

```

Reply:
@mikehardy I've added the updated docs in a11464e08ee4df5c401a019f40a3d6102f8c94aa 😄